### PR TITLE
feat: ✨ shell themes for fzf and skim

### PR DESCRIPTION
**Pull request by tkapias**
Themes for [fzf](https://github.com/junegunn/fzf) and [skim](https://github.com/skim-rs/skim).

image

### CODE REVIEW

File: fzf-skim/.bashrc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+,prompt:#ff5fff\
+,header:#ffd392\
+,info:#bfdaff\
+,cursor:#ffe8c8\
+,selected:#ff3600\
+,spinner:#bfdaff\
+,border:#36363a'\
+"
+export FZF_DEFAULT_OPTS="$FZF_DEFAULT_OPTS \
+--pointer='>' \
```
Comment (fenetikm) [inline fzf-skim/.bashrc:24]: probs shouldn't override this and the next line, just the colours.

File: fzf-skim/.bashrc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+,query_bg:#020221\
+,prompt:#ff5fff\
+,header:#ffd392\
+,info:#bfdaff\
+,cursor:#ffe8c8\
+,selected:#ff3600\
+,spinner:#bfdaff\
+,border:#36363a'\
+"
+export FZF_DEFAULT_OPTS="$FZF_DEFAULT_OPTS \
```
Comment (fenetikm) [inline fzf-skim/.bashrc:23]: think it's better to put this into it's own file, say `fzf/fzf.sh`

File: fzf-skim/.bashrc [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# Falcon color themes for fzf and skim
```
Comment (fenetikm) [inline fzf-skim/.bashrc:1]: think these comments are redundant - also put this into its own file at `skim/skim.sh`

**Review by fenetikm — CHANGES_REQUESTED** [review]
Looks good - just a couple of things to clean up please :)

**Comment by tkapias** [comment]
I separated fzf and skim in 2 folders and remove --pointer and --marker.

**Comment by fenetikm** [comment]
Thanks, appreciate it :)

**Pull request merged**